specs/users.sdk.swagger.json

```diff
@@ -1117,12 +1114,12 @@
       "properties": {
         "id": {
           "type": "string",
-          "format": "int64",
           "description": "The FullStory assigned user ID"
         },
         "uid": {
           "type": "string",
           "description": "The application-specific ID you've given to the user",
+          "title": "TODO(sabrina): I think these should be StringValue: nil(noop) is different from \"\"(update to empty string)",
```

Comment:
nit: it is just the sdk swagger, but shouldn't have TODOs showing up as titles

Reply:
oof good catch!. I reverted the swagger updates and just fixed the file name, since swagger changes are really orthogonal to this PR.